CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Geoserver connection library has been removed and new connection object has been added [#190](https://github.com/IN-CORE/incore-services/issues/190)
 - Semantics should display description instead of title [#206](https://github.com/IN-CORE/incore-services/issues/206)
 
+### Fixed
+- Semantic services endpoint returns correct ID [#206](https://github.com/IN-CORE/incore-services/issues/206)
```

Comment:
This should be issue 227

Reply:
haha i'll update that for you @ywkim312 . I'm in a hurry!

Reply:
Updated